playbooks/luna.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: all
+  become: true
+  vars:
+    foreman_installer_options:
```

Comment:
This should be ```foreman_installer_options_internal_use_only```

Reply:
The Katello meta role does not allow me to override that. Hence why I opted for this given this is specifically attempting to set certain installer variables for it's purpose. The other option is I just don't use the Katello role itself and re-implement it here :/

Reply:
In the `luna-devel` playbook you import another playbook. Would that work here as well?